packages/react/src/components/LocaleModal/LocaleModalCountries.js

```diff
@@ -29,6 +29,7 @@ const LocaleModalCountries = ({
   ...modalLabels
 }) => {
   useEffect(() => {
+    const localeList = document.querySelector(`.${prefix}--locale-modal__list`);
```

Comment:
Probably we can grab this element via React ref?

Reply:
Thanks Akira! Just changed. Let me know if that is better!